remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/SecurityActions.java

```diff
@@ -22,14 +19,6 @@
  */
 final class SecurityActions {
```

Comment:
Are you planning to keep this class because of `getSerializationContext`? 

The other methods can be removed since they already exist in the new `org.infinispan.security.actions.SecurityActions`.